docs/docs/code/nodejs/README.md

```diff
@@ -34,9 +26,13 @@ JavaScript is one of the [most used](https://insights.stackoverflow.com/survey/2
 You can add any Node.js code in the editor that appears. For example, try:
 
 ```javascript
-console.log("This is Node.js code");
-this.test = "Some test data";
-return "Test data";
+defineComponent({
+  async run({ steps, $ }) {
+    console.log("This is Node.js code");
+    this.test = "Some test data";
```

Comment:
`this.test` won't work. `$.export("test", "Some test data")` is the equivalent.

Reply:
Ugh of course. On it.